src/wp-admin/security.php

```diff
@@ -26,6 +51,106 @@
         'https://forums.classicpress.net/security'
     ); ?></p>
   </div>
+
+<?php
+        break;
+    case 'developers':
+?>
+
+  <div id="poststuff">
+    <div id="post-body" class="metabox-holder columns-2">
+      <div id="post-body-content">
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'Overview' ); ?></span></h2>
+            <div class="inside">
+              <div>
+                <p>The Security page is the first visible step in improving the overall approach to security in ClassicPress. Its pupose is to solve two related problems: <strong>discovery</strong> and <strong>auditing</strong>.</p>
+                <p>Most plugins are organised around their core purpose; after all, that purpose is the reason they were installed. Unfortunately this leads to poor discoverability of security-related settings - they may be in there somewhere, there may be none at all - without looking through everything there&lsquo;s no way to know.</p>
+                <p>Currenty, auditing the overall security profile of a ClassicPress site is impractical if there are more than a few plugins. Having all the security-related settings from all the plugins in one place means those settings can be autidted far more easily, as the time taken will be proportional to the number of settings, not the number of places to look for those settings.</p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'API' ); ?></span></h2>
+            <div class="inside">
+              <p>Notes: I left $capability for consistency; we should probably force it to 'manage_options' or just drop it completely - discuss.</p>
+              <!-- HTML generated using hilite.me -->
```

Comment:
Which color scheme was used on hilite.me? Were there any modifications made to the resulting inline CSS?

Reply:
default, no.

Reply:
It looks like the opening `<?php` tag needed to get hilite.me to colorize PHP code was removed. Let's document that, the name of the default color scheme, and any other steps needed to reproduce this example using hilite.me.